.github/ISSUE_TEMPLATE/epic.yml

```
@@ -1,6 +1,7 @@
-name: Epic
-description: (for conda-core team only) A collection of smaller related tickets
-labels: [epic]
+name: "[private] Epic"
+description: "[for conda team only] A collection of related tickets."
```

I don't agree with this being private or just limited to conda. That we only work on epics at Anaconda is more of a bug than a feature to me and it's worth allowing others in the community to open new ones to encourage community contributions.

```suggestion
name: "Epic"
description: "A collection of related tickets."
```

---

But I agree that a better term than "private" could be used here to indicate the unique status of this form. How about "planning" (e.g. `[planning] Epic`) and then including a descriptive blurb in the form? This could be reused for future planning forms we may desire (e.g. user stories, spikes, etc).